src/ui/controls/SwitchControl.tsx

```diff
@@ -39,7 +42,7 @@ export class SwitchControl extends React.Component<IBooleanControlProps, void> {
 
   /** @override */
   componentDidUpdate() {
-    let materialSwitch = this.refs[CSS.RMX_SWITCH]["MaterialSwitch"];
+    let materialSwitch = this.switchControl["MaterialRadio"];
```

Comment:
And yes, MDL attaches these data elements to the controls, and each one has a specific way to trigger events programmatically on it.